#The form ${!FLAG} throws bad substitution errors, converting to a posix complient form fixs it, and allows second pass generation to work
--- a/configure.in
+++ b/configure.in
@@ -249,7 +249,7 @@
 else
   eval $FLAG=no
 fi])
-if test ${!FLAG} = yes ; then
+if test [ "$FLAG" != yes ] ; then
   CFLAGS="$1 $CFLAGS"
 fi
 ])
